src/gameboy/include/gameboy/gameboy.h

```diff
@@ -95,6 +101,14 @@ class Gameboy {
 	 */
 	Gameboy(std::string rom_path);
 
+	/**
+	 * @brief Construct a new Gameboy object, use this only in case of a
+	 * debugger set true for recording the controls
+	 *
+	 * @param rom_path Path to ROM File
+	 */
+	Gameboy(std::string rom_path, bool is_recording);
+
```

Comment:
Why a second override? Let this be the only implementation, and give `isrecording` a default of false.

Reply:
Yeah cool. Will implement this!